.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish Python distributions to PyPI and TestPyPI
+
+on: push
```

Comment:
This looks like a mistake? Surely you don't want to publish to testpypi with every single push?

Reply:
Well, it sure was good for testing this. But, at the moment, no need for this.